serverless/knative-serving/config-applications/multi-container-support-for-serving.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="multi-container-support-for-serving"]
+= Multi-container support for Serving
+:context: multi-container-support-for-serving
+
+You can deploy a multi-container pod using a single Knative service. This is useful for separating application responsibilities into smaller, specialized parts.
```

Comment:
```suggestion
You can deploy a multi-container pod by using a single Knative service. This procedure is useful for separating application responsibilities into smaller, specialized parts.
```
- IBM Style word usage:
![Screenshot 2023-04-06 at 10 28 54 AM](https://user-images.githubusercontent.com/102675557/230408502-3b8a2969-b5f0-4211-9b3e-25a467a071b9.png)

- [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references):  "Avoid sentences in which a pronoun can refer to more than one noun or in which a pronoun refers to an unstated noun."

Reply:
@lahinson Fixed, although used "method" instead of "procedure", it is more precise here.